src/main/java/milutrock/MiluTrock.java

```diff
@@ -17,7 +21,7 @@ public MiluTrock(String path) {
         storage = new Storage(path, parser);
     }
 
-    public void run() {
+    private void run() {
         ui.printBanner();
 
         storage.loadTasks();
```

Comment:
The comments added are helpful in understanding the purpose of each method. However, ensure that the comments stay consistent with the method's name and purpose. For example, the comment for the `loadTasks` method is good but could be improved by explicitly stating that it reads tasks from a file.